src/ontology/ro-edit.owl

```diff
@@ -6550,9 +6550,9 @@ AnnotationAssertion(rdfs:label obo:RO_0017001 "device utilizes material"@en)
 AnnotationAssertion(obo:IAO_0000112 obo:RO_0017008 "biobank organization owns a specimen")
 AnnotationAssertion(obo:IAO_0000112 obo:RO_0017008 "hospital owns a laboratory facility")
 AnnotationAssertion(Annotation(dc:source "OBIB:0000735") obo:IAO_0000115 obo:RO_0017008 "A primitive relation that holds between entities x and y in which y is at x’s full disposal.")
-AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0001-9625-1899>)
-AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1834-3856>)
-AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1909-9269>)
+AnnotationAssertion(terms:contributor obo:RO_0017008 <https://orcid.org/0000-0001-9625-1899>)
+AnnotationAssertion(terms:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1834-3856>)
+AnnotationAssertion(terms:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1909-9269>)
 AnnotationAssertion(dc:description obo:RO_0017008 "This primitive relation is the foundation of the owner’s right to have the owned entity at his/her full disposal.")
```

Comment:
Because this PR was created before we changed the prefix from `dc` to `terms`, you must replace all cases from `dc` to `terms`.

Reply:
@anitacaron Sorry about that. I thought I got them all. Just did another PR.

Reply:
@CDowland, no worries, thanks for the changes